packages/vm/test/api/customChain.spec.ts

```diff
@@ -50,10 +50,10 @@ const genesisState: GenesisState = {
   [contractAddress]: accountState,
 }
 
-const common = new Common({
+// @ts-ignore PrefixedHesString type is too strict
```

Comment:
Was this PrefixedHexString comment something one could not "fix on the fly"? 🤔 

If something is "structurally" too strict, we should maybe un-stricten it?

Reply:
When we take JSON and pass it directly to the various constructors that have the `PrefixedHexString` type, Typescript now complains because we're passing in `string` s. We either need to loosen the type on the opts that can be passed to the various `*Data` constructors or else type check all these strings ahead of time to confirm they're prefixed correctly.